mainwindow.cpp

```diff
@@ -871,15 +851,15 @@ void MainWindow::showJohnFinished()
     checkNToggleActionsLastSession();
     // When John stops we need to stop timer and to look status last
     // time.
-    // TODO: currently if john crashed we do not call john.
+    // Currently if john crashed we do not call john.
```

Comment:
I don't know. Maybe what Aleksey meant with this TODO is that currently showJohnFinished isn't called when John crashed so callJohnShow() won't be called and the tableview of .pot won't be updated ...